doctr/datasets/loaders/recognition/core.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import os
+import json
+import tensorflow as tf
+import numpy as np
+import cv2
+from typing import Tuple, List
+
+__all__ = ["RecognitionDataGenerator"]
+
+
+def load_annotation(
+    labels_path: str,
+    img_name: str,
+) -> str:
+    """Loads recognition annotation (character sequence) for an image from a json file
+
+    Args:
+        labels_path: path to the json file containing annotations
+        img_name: name of the image to find the corresponding label
+
+    Returns:
+        A string (character sequence)
+    """
+    with open(labels_path) as f:
+        labels = json.load(f)
+
+    if img_name not in labels.keys():
+        raise AttributeError("Image is not in referenced in label file")
+
+    return labels[img_name]
+
+
+class RecognitionDataGenerator(tf.keras.utils.Sequence):
+    """Data loader for recognition model
+
+    Args:
+        input_size: size (h, w) for the images
+        images_path: path to the images folder
+        labels_path: pathe to the json file containing all labels (character sequences)
+        batch_size: batch size to train on
+        suffle: if True, dataset is shuffled between each epoch
+        std_rgb: to normalize dataset
+        std_mean: to normalize dataset
+
+    """
+    def __init__(
+        self,
+        input_size: Tuple[int, int],
+        images_path: str,
+        labels_path: str,
+        batch_size: int = 64,
+        shuffle: bool = True,
+        std: Tuple[float, float, float] = (0.299, 0.296, 0.301),
+        mean: Tuple[float, float, float] = (0.694, 0.695, 0.693),
```

Comment:
mind inversing the order please? (usually mean, then std in most frameworks/libraries)